src/main/java/nextstep/session/domain/ImageType.java

```diff
@@ -0,0 +1,17 @@
+package nextstep.session.domain;
+
+import nextstep.session.InvalidImageConditionsException;
+
+public enum ImageType {
+    gif, jpg, jpeg, png, svg;
+
+    public static boolean validateType(String input) throws InvalidImageConditionsException {
+        try {
+            ImageType.valueOf(input.trim().toLowerCase());
+            return true;
+        } catch (IllegalArgumentException e) {
+            return false;
+
+        }
```

Comment:
```suggestion
    public static boolean validateType(String input) {
        return Arrays.stream(ImageType.values())
            .anyMatch(imageType -> imageType.isSame(input.toLowerCase()));
    }
```

구현하신거랑 특별한 장점이 무엇이냐고 한다면 솔찍히 잘 모르겠지만,
저는 이렇게 표현할것 같아요 😄